log/logging.go

```diff
@@ -413,6 +426,12 @@ func (s *logger) logMessage(level LogLevelName, description string, keysAndValue
 		}
 	}
 
+	// hack in caller stats
+	if defaultStackTrace {
+		_, fn, line, _ := runtime.Caller(3)
```

Comment:
I'm not 100% sure this will always be a depth of 3. For instance, `log.Info(...)` and `log.DefaultLogger.Info(...)` would have a depth of 3 and 2 respectively, no?

Reply:
@kevin-cantwell I've tested it a bit.  I created a package with had an Info call in it, That package was imported by another package, which also had a log line.  I created a main and spun up log lines with gorountines and without.  It all seemed okay..

Also I saw 3 being used here as well

https://stackoverflow.com/questions/35212985/is-it-possible-get-information-about-caller-function-in-golang

If you have any thoughts let me know. I don't see to many great ways of doing this

Reply:
Also this is something that was solved by the standard libraries implementation of log

https://golangcode.com/add-line-numbers-to-log-output/

Reply:
@dimamix and @kevin-cantwell  I think that we should drop Golog's implementation in favor for integrating with Go's build in log

https://www.goinggo.net/2013/11/using-log-package-in-go.html